packages/bodiless-core/src/contextMenuForm.static.tsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright © 2019 Johnson & Johnson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useCallback } from 'react';
+import type { ContextMenuFormProps } from './Types/ContextMenuTypes';
+
+export const ContextMenuForm = () => <></>;
```

Comment:
Here again why create this extra compnoent and not just use `Fragment`?

Reply:
fixed this as well